load_tests/load_test.py

```diff
@@ -169,11 +255,27 @@ def delete_testing_resources():
     bucket = s3.Bucket(os.environ['S3_BUCKET_NAME'])
     bucket.objects.all().delete()
 
+def get_destination_identifier(throughput, input_logger):
+    # Destination identifier
+    # [log source] ----- (stdout) -> std-{{throughput}}/...
+    #               \___ (tcp   ) -> {{throughput}}/...
+    #
+    # All inputs should have throughput as destination identifier
+    # except stdstream
+    destination_identifier = throughput
+    if (input_logger['name'] == 'stdstream'):
+        destination_identifier = 'std-' + throughput
+    return destination_identifier
+
+def get_destination_identifier_firelens(throughput, input_logger):
+    return 'std-' + throughput
+
 if sys.argv[1] == 'create_testing_resources':
     create_testing_resources()
 elif sys.argv[1] == 'ECS':
+    delete_testing_data()
```

Comment:
I saw this function in line 157 so I guess it is already called before every run? do we still call it here?

Reply:
I think this is safe to remove! good catch